packages/back-end/src/controllers/datasources.ts

```diff
@@ -437,6 +439,34 @@ export async function putDataSource(
   >,
   res: Response
 ) {
+  const userId = req.userId;
+
+  if (!userId) {
+    res.status(403).json({
+      status: 403,
+      message: "User not found",
+    });
+    return;
+  }
+
+  const user = await getUserById(userId);
+
+  if (!user) {
+    res.status(403).json({
+      status: 403,
+      message: "User not found",
+    });
+    return;
+  }
+
+  const userObj: Omit<
+    ExpandedMember,
```

Comment:
Why not make this an `AuditUserLoggedIn`

Reply:
Thank you! I could not find an available type that sufficed!